cmd/sync/sync.go

```diff
@@ -518,3 +634,17 @@ func combinedOutput(c *exec.Cmd) (string, error) {
 	}
 	return string(out), nil
 }
+
+func createCommitMessageSnippet(message string) (string, error) {
+	s := bufio.NewScanner(strings.NewReader(message))
+	s.Scan()
+	if err := s.Err(); err != nil {
+		return "", err
+	}
+
+	snippet := s.Text()
```

Comment:
Is the scanner just advancing one line? If so this can be simplified with something like:

```suggestion
        snippet := message
        if i := strings.IndexByte(snippet, '\n'); i >= 0 {
                snippet = snippet[i+1:]
        }
```

Reply:
I'm concerned about running into `\r\n` rather than just `\n`. *Maybe* not necessary with output from Git commands, but I'm not sure. (I used this approach earlier to read from a file, where `\r\n` handling is more obviously needed.)

But, I'll switch to `strings.IndexAny(snippet, "\r\n")`, which is still simpler than the scanner. (The behavior I'm looking for is `[:i]`, rather than `[i+1:]` though. Which the scanner doesn't exactly make clear! 😄)